### DISCUSSION TITLE
[Feature Requests] Multiple character delimiter needs to be supported

### DISCUSSION THREAD

**Discussion by liang8283** [apache/cloudberry#200] [Ideas / Feature Requests]
### Description

In hashdata 3x, this feature has been implemented as following:

1. create a teset file.
cat 1.txt
aaa|@|bbb

2. create a table.   
create table t1 (c1 text, c2 text);

3. create external table to specify multiple characters '|@|' as the delimeter.   
CREATE EXTERNAL TABLE t1_ext (LIKE t1)
LOCATION ('gpfdist://192.168.177.161:8081/1.txt')
FORMAT 'CUSTOM' (FORMATTER=delimiter_in,entry_delim='|@|',line_delim=E'\n',null='',encoding='utf8');

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

**Comment by tuhaihe** [comment]
Hi @my-ship-it, how about this feature request for CloudberryDB? 

Furthermore, Should we create a missing feature list between HashData 3.x and CloudberryDB to ensure a seamless user experience during the migration from HashData 3.x to CloudberryDB? Just my opinion, FYI.
